codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/protocol/ServerProtocolTestGenerator.kt

```diff
@@ -242,14 +249,19 @@ class ServerProtocolTestGenerator(
         writeInline("let output =")
         instantiator.render(this, expectedShape, testCase.params)
         write(";")
-        val operationName = if (expectedShape.hasTrait<ErrorTrait>()) {
-            "${operationSymbol.name}${ServerHttpProtocolGenerator.OPERATION_ERROR_WRAPPER_SUFFIX}"
+        val operationImpl = if (operationShape.errors.isNotEmpty()) {
+            if (expectedShape.hasTrait<ErrorTrait>()) {
+                val symbol = symbolProvider.toSymbol(expectedShape).name
+                "${operationSymbol.name}${ServerHttpProtocolGenerator.OPERATION_OUTPUT_WRAPPER_SUFFIX}::Error(crate::error::${operationSymbol.name}Error::$symbol(output))"
```

Comment:
could be worth adding a private extension function on operationShape to generate this path and clean up the code.

```
private fun OperationShape.serverShapeName()
```

```
${operationShape.serverShapeName()}::Error(...)
${operationShape.serverShapeName()}::Output(...)
etc.
```

If you define it inside of the test generator, you'll have access to the symbolProvider which makes for fairly clean code

Reply:
Yes, this is more or less the plan. Sorry again, I forgot to mark this PR as Draft :(